taskchampion/taskchampion/src/errors.rs

```diff
@@ -12,4 +16,23 @@ pub enum Error {
     /// other irrecoverable error.
     #[error("Local replica is out of sync with the server")]
     OutOfSync,
+    /// A usage error
+    #[error("User Error: {0}")]
+    UserError(String),
```

Comment:
I like the word "Usage" here -- there may not be a "user" involved, but it indicates the usage of some function was incorrect.